profiler/wdt_follow_ajax.rst

```diff
@@ -0,0 +1,40 @@
+.. index::
+    single: Profiling: WDT Auto-update after AJAX Request
+
+How to make the Web Debug Toolbar Auto-upate after AJAX Requests
+================================================================
+
+For single page applications it is often inconvenient to find the profiler
+information for the most recent request. It would be much more convient if it
```

Comment:
typo `convenient`

Reply:
Consider rephrase to:

>It would be more convenient if the toolbar showed the information for the most recent AJAX request instead of the initial page load.